scripts/run_mypy_checks_test.py

```diff
@@ -68,12 +71,39 @@ def mock_popen_failure(
             run_mypy_checks, 'EXCLUDED_DIRECTORIES',
             ['dir1/', 'dir2/'])
 
-        def mock_install_mypy_prerequisites_success():
+        def mock_install_mypy_prerequisites_success(ci): # pylint: disable=unused-argument
             return 0
         self.swap_install_success = self.swap(
             run_mypy_checks, 'install_mypy_prerequisites',
             mock_install_mypy_prerequisites_success)
 
+        def mock_popen_user_prefix_error_call(
+                unused_cmd_tokens, *args, **kwargs):  # pylint: disable=unused-argument
+            class Ret(python_utils.OBJECT):
+                """Return object that gives user-prefix error."""
+
+                def __init__(self):
+                    if '--user' in unused_cmd_tokens:
```

Comment:
Don't call something `unused_*` if you use it! :smile:

Reply:
My bad! Changed it.

Reply:
@DubeySandeep should the linter have caught this?